thirdparty/mupdf/external_fonts.patch

```diff
@@ -93,6 +94,22 @@ index f6951ba2..b1fdd981 100644
 +	return filename;
 +}
 +
++char *
++get_font_file_android(char *name)
++{
++	char *fontdir;
```

Comment:
```C
	const char fontdir[] = "/system/fonts";
	size_t len = sizeof(fontdir) + strlen(name) + 1;	// sizeof on a string literal includes the NUL
	char *filename = malloc(len);
	if (filename == NULL) {
		return NULL;
	}
```

(Fairly minor, this works ;)).

Reply:
I'm pretty sure I tested with what I thought of as a stress test (EPUB) but I guess that just used the Times (or whatever) that's bundled with MuPDF. In any case I didn't have any HTML dictionaries installed on the emulator.

Reply:
You're fine with the basic principle besides this nit? :-P

Reply:
Yep!